### PR TITLE
Add main documentation

### PR DESCRIPTION
**Pull request by SBoudrias**
First draft of our main documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,175 @@
 # AB Testing
+
+AB testing library supporting multi-variance testing with a deterministic algorithm not requiring any complex backend or database.
+
+The segmentation logic is maintained in the AB testing client and based itself on a centralized configuration. The cohort assignment logic is deterministic and follows a simple hash pattern based on the `crc32c` algorithm (`crc32c(userId + crc32c(experimentName)) % 100`)
```
Comment (ZigZagT) [inline README.md]: not sure if the algorithm can be written like this, the `+` is a bit confusing me.
The actual call to crc32c is `crc32.calculate(String(userId), crc32.calculate(experiment)) % 100`, where the second  parameter of `crc32.calculate` is:
> initial optional, Number, initial CRC-32C digest

by default it's 0

Comment (SBoudrias) [reply README.md]: Right! Updated :)

File: packages/ab-testing/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# AB Testing
+
+AB testing library supporting multi-variance testing with a deterministic algorithm not requiring any complex backend or database.
+
+The segmentation logic is maintained in the AB testing client and based itself on a centralized configuration. The cohort assignment logic is deterministic and follows a simple hash pattern based on the `crc32c` algorithm (`crc32c(userId + crc32c(experimentName)) % 100`)
```
Comment (ZigZagT) [inline packages/ab-testing/README.md]: also here :P

**Review by ZigZagT — DISMISSED** [review]
LGTM

File: packages/ab-testing/README.md (hunk 2 above)
Comment (SBoudrias) [reply packages/ab-testing/README.md]: uh oh!

**Review by ZigZagT — APPROVED** [review]

**Pull request merged**